spaceship/integration/portal/certificate_spec.rb

```diff
@@ -0,0 +1,54 @@
+describe Spaceship do
+  describe Spaceship::Portal do
+    describe Spaceship::Portal::Certificate do
+      let(:certificate) { Spaceship::Portal.certificate }
+
+      before(:all) do
+        @certificates = Spaceship::Portal.certificate.all
+      end
+
+      it 'finds certificates on the portal' do
+        expect(@certificates).to_not be_empty
+      end
+
+      it 'fetched certificates have reasonable data' do
+        cert = @certificates.first
+
+        expect(cert.id).to match_apple_ten_char_id
+        expect(cert.name).to_not be_empty
+        expect(cert.status).to eq('Issued')
+        expect(cert.created).to be_kind_of(Time)
+        expect(cert.expires).to be_kind_of(Time)
+        expect(cert.owner_id).to match_apple_ten_char_id
+        expect(cert.type_display_id).to match_apple_ten_char_id
+        expect(cert.can_download).to be(true)
+      end
+
+      it 'certificate creation and revokation work' do
+        # Create a new certificate signing request
+        csr, = certificate.create_certificate_signing_request
```

Comment:
This felt like more of the responsibility of the unit tests, since CSR creation doesn't talk to Apple servers.